src/Sign.Core/IServiceProviderFactory.cs

```diff
@@ -13,5 +13,7 @@ IServiceProvider Create(
             LogLevel logLevel = LogLevel.Information,
             ILoggerProvider? loggerProvider = null,
             Action<IServiceCollection>? addServices = null);
+
+        void AddServices(Action<IServiceCollection> services);
     }
-}
\ No newline at end of file
+}
```

Comment:
[.editorconfig enforces a blank line at the end of the file.](https://github.com/dotnet/sign/blob/8afa68d54e3a7271e2dc5129da790286e91e33aa/.editorconfig#L4)  Are you using an editor that doesn't respect .editorconfig?

The same rule break also happened in several other files in this PR.

Reply:
I'm using VS 2022 Preview. strange